sql/core/src/test/resources/sql-tests/results/explain-aqe.sql.out

```diff
@@ -909,7 +909,7 @@ Execute CreateViewCommand (1)
 Output: []
 
 (2) CreateViewCommand
-Arguments: `spark_catalog`.`default`.`explain_view`, SELECT key, val FROM explain_temp1, false, false, PersistedView, true
```

Comment:
we shouldn't attach v1 catalog name here as `CreateViewCommand` can create temp views.

Reply:
it seems we can distinguish if the target view type of CreateViewCommand is temp ?

Reply:
good point.